Packages/com.unity.inputsystem/InputSystem/Controls/InputControlLayout.cs

```diff
@@ -30,6 +30,31 @@
 ////REVIEW: useStateFrom seems like a half-measure; it solves the problem of setting up state blocks but they often also
 ////        require a specific set of processors
 
+////REVIEW: Can we allow aliases to be paths rather than just plain names? This would allow changing the hierarchy around while
+////        keeping backwards-compatibility.
+
+// Q: Why the heck is there this layout system instead of just new'ing everything up in hand-written C# code?
```

Comment:
Can we say heck in the docs? ;)

Reply:
Why the heck not? :D

Seriously, though, as a non-native speaker, I find it sometimes hard to judge what's offensive/explicit/getting-you-sideways-glances-at-parties and what isn't. I'll go remove it :)

Reply:
Haha. Its not really offensive(I think), just a bit too casual perhaps ;)

Reply:
Nuked :)